server/api.py

```
@@ -0,0 +1,313 @@
+    permission: List = []
+    context_length: Optional[int] = None
+
+class StartRequest(BaseModel):
+    model: str
+    memory_path: str
+
+class Agent:
+    def __init__(
+        self,
+        max_tool_turns: int = 20,
+        memory_path: str = None,
```

Ruff (0.14.1)

Unused method argument: `memory_path`

(ARG002)